ai_ta_backend/vector_database.py

```diff
@@ -1109,14 +1223,14 @@ def get_stuffed_prompt(self, search_query: str, course_name: str, token_limit: i
       String: A fully formatted prompt string.
     """
     try:
-      top_n = 90
+      top_n = 150
       start_time_overall = time.monotonic()
       o = OpenAIEmbeddings() # type: ignore
       user_query_embedding = o.embed_documents(search_query)[0] # type: ignore
       myfilter = models.Filter(
               must=[
                   models.FieldCondition(
-                      key='course_name',

```

Comment:
I think this should stay as `key='course_name'`, that's the new format in Qdrant (already live in prod).